pkg/db/queries.sql

```diff
@@ -3,3 +3,11 @@ INSERT INTO staged_originator_envelopes(payer_envelope)
 	VALUES (@payer_envelope)
 RETURNING
 	*;
+
+-- name: InsertNodeInfo :one
+INSERT INTO node_info(node_id, public_key)
+	VALUES (@node_id, @public_key)
+	RETURNING *;
+
+-- name: SelectNodeInfo :one
+SELECT * FROM node_info WHERE singleton_id = 1;
```

Comment:
These could have been combined into a single query that fetches the existing record when there is a conflict. Unfortunately, it looks like the methods for doing this in Postgres either impact readability significantly or have weird caveats.

I chose to do this the simple way because server startup performance isn't critical here.

Reply:
Why does this need to live in the database at all? Can't the `NodeRegistry` just serve as the source of truth here?

Reply:
Yes, purely so that you get an error if you change either piece of information with the same DB. I imagine it's an easy mistake for partners to make

Reply:
OK. Good to be defensive here. Could definitely see it being an easy mistake to make.